narayana/ArjunaJTA/jta/tests/classes/org/jboss/narayana/rts/TxnHelper.java

```diff
@@ -71,7 +69,7 @@ public static int endTxn(Client client, Set<Link> links) throws IOException {
 
             int sc = response.getStatus();
 
-            EntityUtils.consume((HttpEntity) response.getEntity());
+            response.getEntity();
```

Comment:
And similarly here, we want to be certain that the entity input stream is fully consumed so you need response.readEntity(String.class) here.

Reply:
Thanks, fixed it.